src/services/edge-application-origins-services/create-origin-service.js

```diff
@@ -19,7 +19,7 @@ const adapt = (payload) => {
       name: payload.name,
       origin_type: payload.originType,
       bucket: payload.bucketName,
-      prefix: `/${payload.prefix}`.trim()
+      prefix: payload.prefix.trim() === '' ? '/' : payload.prefix
```

Comment:
Dessa forma não resolveria: `payload.prefix.trim() || '/';` ?

Reply:
sim. otima sugestão, irei incluir

Reply:
Não deveria retornar algo assim:
```suggestion
      prefix: !!payload.prefix.trim() ? '/' : payload.prefix.trim()
```

Reply:
E talvez criar uma `const` que tenha o resultado do `trim()` para não executá-lo repetidas vezes.